packages/ui/src/components/MenuV2/MenuContent.tsx

```diff
@@ -146,12 +150,13 @@ export const Menu = forwardRef(
     )
 
     useEffect(() => {
-      if (isVisible && searchable) {
+      if (isVisible) {
```

Comment:
There is a weird behavior, If I click on the menu then I press Tab, only the second item is focused ? Maybe it is because the first one is implicitly focused


https://github.com/user-attachments/assets/793f62ad-58b4-464b-b920-fcb6130f527e